packages/typespec-python/generator/pygen/preprocess/__init__.py

```diff
@@ -362,7 +362,12 @@ def update_operation(
         yaml_data["groupName"] = self.pad_reserved_words(yaml_data["groupName"], PadType.OPERATION_GROUP)
         yaml_data["groupName"] = to_snake_case(yaml_data["groupName"])
         yaml_data["name"] = yaml_data["name"].lower()
-        yaml_data["name"] = self.pad_reserved_words(yaml_data["name"], PadType.METHOD)
+        if yaml_data.get("isLroInitialOperation") is True:
+            yaml_data["name"] = (
```

Comment:
Make sure the builder request name is same between init lro operation and lro operation to avoid import error for aio.